### PR TITLE
Feat: Image-input

### PR DESCRIPTION
**Pull request by msdeshmukh009**
simple-inputs
disabled inputs
validator-inputs
signup-form

### CODE REVIEW

File: docsite-css/component-display-conatiner.css [hunk 1]
```diff
@@ -24,6 +24,10 @@
   width: 50%;
   margin: auto;
 }
+.component-display-container .form-grp {
+  margin-left: auto;
+  margin-right: auto;
+}
```
Comment (gopalbbharadva) [inline docsite-css/component-display-conatiner.css:28]: Improvment: You give margins like { margin: 0 auto }.

**Review by gopalbbharadva — APPROVED** [review]

**Review by himadri2110 — APPROVED** [review]

File: components/Inputs/inputs.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+.form-field {
+  display: block;
+  width: 100%;
+  padding: 0.65rem 1rem;
+  border-radius: 0.3rem;
+  border: 2px solid var(--secondary-color);
+  outline: none;
+}
+.form-field:focus {
+  box-shadow: 0 0 0 1px var(--primary-color);
```
Comment (mimansha-swarup) [inline components/Inputs/inputs.css]: you can make variable for box-shadow too

**Review by mimansha-swarup — COMMENTED** [review]
except that all looks good

File: components/Images/images.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      <main id="doc-main-content" class="doc-main-content">
+        <div class="component-description">
+          <h1>Images</h1>
+          <p>
+            Lorem ipsum dolor, sit amet consectetur adipisicing elit. Vitae cupiditate illo atque exercitationem, aliquid dolor odit sequi aliquam
+            provident non!
+          </p>
+        </div>
+
+        <div class="componet-variation-description">
```
Comment (MarkVed17) [inline components/Images/images.html:60]: a typo here (component*)

File: components/Images/images.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+        <code class="code-display-container">
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28229%2C236%2C238%2C1%29&t=one-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=20px&ph=52px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522responsive-img%2522%2520src%253D%2522%252Fassets%252Ftavin-dotson-WC4IWN3-fSo-unsplash.jpg%2522%2520alt%253D%2522flower%2520sample%2520image%2522%2520%252F%253E"
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+        </code>
+
+        <div class="componet-variation-description">
```
Comment (MarkVed17) [inline components/Images/images.html:80]: typo here

**Review by MarkVed17 — APPROVED** [review]
Just fix those typos & you will be good to proceed.

File: docsite-css/component-display-conatiner.css (hunk 1 above)
Comment (msdeshmukh009) [reply docsite-css/component-display-conatiner.css:28]: Ohh that is right. I will do that 😄

File: components/Inputs/inputs.css (hunk 2 above)
Comment (msdeshmukh009) [reply components/Inputs/inputs.css]: yes, I always forget that. Thanks!

File: components/Images/images.html (hunk 3 above)
Comment (msdeshmukh009) [reply components/Images/images.html:60]: Thanks for pointing it out will need to make the changes in lots of places now 😅

**Pull request merged**